src/NodeApi/JSPropertyDescriptor.cs

```diff
@@ -68,6 +79,22 @@ namespace Microsoft.JavaScript.NodeApi;
         Data = data;
     }
 
+    /// <summary>
+    /// Creates a property descriptor with a value.
+    /// </summary>
+    public static JSPropertyDescriptor Property(
```

Comment:
I find the new name to be less descriptive than the previous `ForValue` name. I would suggest to use the same term as in the JS docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty
The use terms "data descriptor" vs "accessor descriptor".
E,g, we can do the following renames:
- `Property` -> `DataProperty`
- `Accessor` -> `AccessorProperty`
- `Function` -> `Method`

Reply:
Yeah I debated a few different options there.

I changed `ForValue` because it seemed awkward when the other methods (`Accessor` and `Function`) don't begin with `For`. The reason for the `For` prefix is that there is already a property named `Value`, so there cannot be a method with the same name.

`DataProperty` and `AccessorProperty` are good ideas. But `Method` won't work because there is already a property of the class named `Method`.

Alternatively we could just make these all overloads of the `JSPropertyDescriptor` constructor. It would be more concise, but less obvious what kind of descriptor is being created.

Reply:
We can keep the `Function` then. IMHO, the static methods are more descriptive than constructors.

Reply:
Done.